### ISSUE TITLE
SWIG 4.0 appears to break the kiva.agg backend

### ISSUE THREAD

**Issue by jwiggins** [enthought/enable#360]
See #359 for the full saga. For now, the solution is to just build with 3.0.x.

**Comment by yves-surrel** [comment]
I confirm the problem and the short-term fix

**Comment by swt2c** [comment]
Any progress on the long-term fix?  :)  Debian has upgraded to swig 4.0 and we're seeing problems there as well.  It seems that `_swig_setattr` has been removed as well.

```
...
test_minimum_vertical (enable.tests.qt4.scrollbar_test.ScrollBarTest) ... Traceback (most recent call last):
  File "/build/1st/python-enable-4.8.1/.pybuild/cpython3_3.8_enable/build/enable/qt4/base_window.py", line 213, in paintEvent
    self.handler.paintEvent(event)
  File "/build/1st/python-enable-4.8.1/.pybuild/cpython3_3.8_enable/build/enable/qt4/base_window.py", line 65, in paintEvent
    self._enable_window._paint(event)
  File "/build/1st/python-enable-4.8.1/.pybuild/cpython3_3.8_enable/build/enable/abstract_window.py", line 462, in _paint
    self._gc = self._create_gc(size)
  File "/build/1st/python-enable-4.8.1/.pybuild/cpython3_3.8_enable/build/enable/qt4/image.py", line 20, in _create_gc
    gc = GraphicsContext((size[0]+1, size[1]+1), pix_format=pix_format,
  File "/build/1st/python-enable-4.8.1/.pybuild/cpython3_3.8_enable/build/kiva/agg/__init__.py", line 38, in __init__
    GraphicsContextArray.__init__(self, pixel_map.bmp_array,
  File "/build/1st/python-enable-4.8.1/.pybuild/cpython3_3.8_enable/build/kiva/agg/agg.py", line 859, in __init__
    _swig_setattr(self, GraphicsContextArray, 'this', obj)
NameError: name '_swig_setattr' is not defined
Aborted
```

**Comment by lateralguy** [comment]
I was facing a similar issue after rolling back the packages to the following version:
enable = 4.8.0
swig = 3.0.12
The issue was resolved.
